source/docs/articles/sites/migrate.md

```diff
@@ -73,6 +73,8 @@ Import code, database, and files after creating the site using a combination of
 ### Create Single-file Archives
 Migrations using the importer, for sites that can be packaged with a total archived size less than 500MB are able to use single-file archives during the import process. Users can create these archives with [drush](/docs/articles/sites/migrate/export-an-existing-drupal-site#create-archive-using-drush) for Drupal sites and  [Plugins](/docs/articles/sites/migrate/export-an-existing-wordpress-site#export-wordpress-via-plugins) for WordPress.
 
+Drupal users can create single-file archives for sites hosted elsewhere or locally with the [Backup and Migrate](https://www.drupal.org/project/backup_migrate) module. When creating the archive, choose to backup your **entire site (code, files, & DB)**. You can choose to download the archive and/or send it to NodeSquirrel for a publicly accessible URL. 
```

Comment:
Let's just list it in the paragraph above (BM or Drush) and link out to the longer set of instructions in the export drupal sites doc.